docs/tutorials/upgrading-ev3dev-from-command-line.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Upgrading Ev3dev from the Command Line
+subject: System
+---
+
+{% include icon.html type="warning" %}
+By upgrading ev3dev you are also upgrading the drivers, so there may be changes that break user software. 
```

Comment:
Saying "upgrading ev3dev" is ambiguous. What is "ev3dev" exactly? If you want to make a statement about upgrading drivers, then saying "upgrading the kernel package" would make more sense because upgrading the kernel is specifically what upgrades the drivers.